packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/stats_table.dart

```diff
@@ -142,32 +142,47 @@ class _StatsTableState extends State<StatsTable> {
       _RetainedSizeColumn(),
     ];
 
-    if (!widget.sorting.initialized) {
-      widget.sorting
+    if (!_item.sorting.initialized) {
+      _item.sorting
         ..direction = SortDirection.descending
         ..columnIndex = _columns.indexOf(_shallowSizeColumn)
         ..initialized = true;
     }
   }
 
+  HeapStatsRecord? get _selectedRecord =>
+      _item.statsToShow.recordsByClass[widget.controller.selectedClass.value];
```

Comment:
we can set a notifier with the selected record in the controller (from controller.setSelectedClass)